corehq/apps/app_manager/views/download.py

```diff
@@ -311,13 +314,17 @@ def download_profile(request, domain, app_id):
     See ApplicationBase.create_profile
 
     """
+    if not request.app.copy_of:
+        make_async_build.delay(request.app)
     return HttpResponse(
         request.app.create_profile()
     )
```

Comment:
i might be missing some context, but does this task need to finish before we return the download response, or do we expect the returned data to sometimes be a build behind?